lib/views/widgets/progress_bar.dart

```diff
@@ -0,0 +1,144 @@
+import 'package:flutter/material.dart';
+import 'package:hooks_riverpod/hooks_riverpod.dart';
+import 'package:sun_flutter_capstone/consts/global_style.dart';
+
+class ProgressBar extends StatefulHookConsumerWidget {
+  double progress;
+  String? label;
+  final Color barColor;
+  final Color bgColor;
+  final double barSize;
+  final String barFontFamily;
+  final double barFontSize;
+  final double barRadius;
+
+  ProgressBar({
+    Key? key,
+    this.progress = 0,
+    this.label,
+    this.barColor = ProgressBarTheme.barColor,
+    this.bgColor = ProgressBarTheme.bgColor,
```

Comment:
Can you drop here the screenshot, where specifically this color is being used, I cant pin point it in the screen